cluster/centos/config-build.sh

```diff
@@ -29,7 +29,7 @@ FLANNEL_VERSION=${FLANNEL_VERSION:-"0.6.1"}
 ETCD_VERSION=${ETCD_VERSION:-"3.0.9"}
 
 # Define k8s version to use.
-K8S_VERSION=${K8S_VERSION:-"1.3.7"}
+K8S_VERSION=$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)
```

Comment:
The old version didn't start with a 'v' character. Have you tested that the rest of the script still works when passing the modified version string?